lib/src/base-styles/_nx-pagination.scss

```diff
@@ -7,6 +7,11 @@
 @use '../scss-shared/_nx-pagination-variables';
 @use '../scss-shared/nx-text-helpers';
 
+%btn-focus-outline {
+  outline: 2px solid var(--nx-color-interactive-border-focus);
+  outline-offset: -5px;
```

Comment:
It looks like it's more accurate if the offset is -5px?
Right now the focus "border" between the outline and the rest of the fill is 3px when it is 2px in Sketch.
Might need to double check this!

<img width="455" alt="Screenshot 2023-03-10 at 2 21 48 PM" src="https://user-images.githubusercontent.com/10160030/224407117-3ee781e8-fbaa-4607-b0ed-6535e16fc275.png">

Reply:
> It looks like it's more accurate if the offset is -5px? Right now the focus "border" between the outline and the rest of the fill is 3px when it is 2px in Sketch. Might need to double check this!
> 
Thanks for finding that! I changed it to -4px (I think in your comment you meant to write -4?), which will a 2px blue 'border' when focused. [ce94da2](https://github.com/sonatype/sonatype-react-shared-components/pull/1004/commits/ce94da20784860b3e993604734369b6b63f755c0)

Reply:
> > It looks like it's more accurate if the offset is -5px? Right now the focus "border" between the outline and the rest of the fill is 3px when it is 2px in Sketch. Might need to double check this!
> 
> Thanks for finding that! I changed it to -4px (I think in your comment you meant to write -4?), which will a 2px blue 'border' when focused. [ce94da2](https://github.com/sonatype/sonatype-react-shared-components/pull/1004/commits/ce94da20784860b3e993604734369b6b63f755c0)

Oh ops yup -4 😸 , looks good! Thanks Emily!